### PR TITLE
Reset dersired state after each test

### PR DESCRIPTION
**Pull request by qinqon**

### CODE REVIEW

File: test/e2e/simple_bridge_and_bond.go [hunk 1]
```diff
@@ -215,6 +219,7 @@ var _ = Describe("NodeNetworkState", func() {
 				for _, node := range nodes {
 					interfacesNameForNode(node).ShouldNot(ContainElement("bond1"))
 				}
+				resetDesiredStateForNodes()
```
Comment (phoracek) [inline test/e2e/simple_bridge_and_bond.go]: Can't we do it in module-wide (or even better, package-wide) AfterEach?

Comment (qinqon) [reply test/e2e/simple_bridge_and_bond.go]: well we don't control what afterEach come last..., prefer to do this way to be sure that this is the last thing is called.

Comment (phoracek) [reply test/e2e/simple_bridge_and_bond.go]: Fine with me. Would you just add a TODO to resetDesiredStateForNodes mentioning that we should drop it once Policy cleans up the state?

**Comment by phoracek** [comment]
ci test please

**Comment by phoracek** [comment]
ci test please

**Pull request merged**